<launch>
    <arg name="veh" doc="Name of vehicle. ex: megaman"/>
    <arg name="config" default="baseline" doc="Specify a config."/>
    <arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman." />
    
    <arg name="pkg_name" value="veh_coordinator"/>
    <arg name="node_name" default="simple_coordinator_node"/>
    <arg name="verbose" default="false" />

    <arg name="intersectionType" default="trafficLight" />

    <group ns="$(arg veh)">        
        <node  name="$(arg node_name)" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen" required="true">
            <rosparam command="load" file="$(find duckietown)/config/$(arg config)/veh_coordinator/$(arg node_name)/$(arg param_file_name).yaml"/>
            <param name="verbose" value="$(arg verbose)"/>
            <param name="intersectionType" value="$(arg intersectionType)"/>
        </node>
    </group>

    <!-- Publications -->

    <!-- Subscriptions -->

</launch>
